indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -4170,7 +4167,7 @@ public SeekableStreamSupervisorIOConfig getIoConfig()
     return ioConfig;
   }
 
-  @Override
+
   public void checkpoint(int taskGroupId, DataSourceMetadata checkpointMetadata)
```

Comment:
Please add the javadocs originally present in `Supervisor` interface.

Reply:
Done, moved the javadocs too.